apps/website/src/pages/admin/unauthorized.tsx

```diff
@@ -0,0 +1,28 @@
+import type { NextPage } from "next";
+
+import Meta from "@/components/content/Meta";
+import Section from "@/components/content/Section";
+import Heading from "@/components/content/Heading";
+
+const UserUnauthorizedPage: NextPage<SigninPageProps> = () => {
+  return (
+    <>
+      <Meta
+        title="Unauthorized"
+        description="You are not authorized to view this page."
+      />
```

Comment:
I'd say we should noindex this page and then we also don't need a description:

```suggestion
      <Meta title="Unauthorized" noindex />
```